diff --git a/lib/options-manager.js b/lib/options-manager.js
index 7d7b157321e0f8d1f7cd2020479826449423b36b..ff8dfeccbdd9dbb647ffe51619eab1fd8331f62b 100644
--- a/lib/options-manager.js
+++ b/lib/options-manager.js
@@ -353,12 +353,6 @@ const buildXOConfig = options => config => {
 		}
 	}
 
-	if (options.nodeVersion) {
-		config.baseConfig.rules['n/no-unsupported-features/es-builtins'] = ['error', {version: options.nodeVersion}];
-		config.baseConfig.rules['n/no-unsupported-features/es-syntax'] = ['error', {version: options.nodeVersion, ignores: ['modules']}];
-		config.baseConfig.rules['n/no-unsupported-features/node-builtins'] = ['error', {version: options.nodeVersion}];
-	}
-
 	if (options.space && !options.prettier) {
 		if (options.ts) {
 			config.baseConfig.rules['@typescript-eslint/indent'] = ['error', spaces, {SwitchCase: 1}];
